pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -240,7 +240,8 @@ rec {
       mkdir -p $dropinDir/features
       unzip ${srcFeature} -d $dropinDir/features/
       mkdir -p $dropinDir/plugins
-      cp -v ${srcPlugin1} ${srcPlugin2} $dropinDir/plugins/
+      cp -v ${srcPlugin1} $dropinDir/plugins/''${srcPlugin1#*-}
```

Comment:
Without the single quotes nix does not parses correctly the shell substitution variable `${srcPlugin1#*-}`.
The idea is to remove the hash code that nix automatically adds to sources from the beginning of the file name.
If it was written directly in a shell script, the double single quotes should not be there. But it is written inside a double single quotes nix string and is subject to nix substitution rules.
This solution was pointed to me in the IRC channel.